packages/canon/css/components.css

```diff
@@ -584,7 +584,7 @@
   margin: .375rem 1rem;
 }
 
-.canon-Link {
+a.canon-Link {
```

Comment:
Seems unfortunate to have `a` in there, it's specific enough as is right? I'm thinking for some weird situations is someone wants to render a disabled link as a `span` or w/e. More generally it seems like a good pattern to keep tags away from component styles

Reply:
Because we set the `a` tag globally, if we don't do this it will not override it.

Reply:
That is unfortunate indeed but I'm not sure we can avoid it unfortunately.

Reply:
I found a better way to set global styles to avoid having to specify the `a` tag in class names using `:where(a)`